SPECS/hvloader/hvloader.spec

```diff
@@ -0,0 +1,62 @@
+%define  debug_package %{nil}
+%define  name_github   HvLoader
+%define  edk2_tag      edk2-stable202211
+Summary:        HvLoader.efi is an EFI application for loading an external hypervisor loader.
+Name:           hvloader
+Version:        1.0.1
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Applications/System
+URL:            https://github.com/microsoft/HvLoader
+# Hvloader (Source1) must be unpacked and built in an edk2 environment (Source0). 
+Source0:        https://github.com/tianocore/edk2/archive/refs/tags/%{edk2_tag}.tar.gz#/edk2-submodules-%{edk2_tag}.tar.gz
+Source1:        https://github.com/microsoft/HvLoader/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source2:        target.txt
```

Comment:
As discussed, let's see if it makes sense to place this in the github repo itself and consume it from there.

Reply:
Let us also look into whether we can get rid of this file and specify the parameters in the file e.g. as env variables? Just a thought

Reply:
It gets messy if we don't provide a target.txt file. 

![image](https://user-images.githubusercontent.com/37827103/235801198-cbff594d-91b4-4769-b762-7f53f2f38058.png)

The way I see it, we can either:
- Keep target.txt as is, makes it easier to modify and duplicate for ARM later
- Use variables in the spec file and sed them into Conf/target.txt
- Check our target.txt into hvloader's codebase. Have the spec copy the file from hvloader's source tar into Conf/target.txt. This is not ideal because others might want to try building the component for their system/architecture. The target.txt describes Mariner's scenario specifically.